.changes/v1.15/BUG FIXES-20251121-183045.yaml

```diff
@@ -0,0 +1,5 @@
+kind: BUG FIXES
```

Comment:
Could this be considered a breaking change?

I'd argue not, as the old behaviour was misleading and incorrect, though someone could have built a system that relied on the command not failing somewhere.